src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -253,32 +291,46 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
       response.sendError(400, e.getMessage());
       return;
     }
-    String position = putRequest.getPosition();
-    Job selectedPosition = Job.valueOf(Job.class, position);
+
     List<ScheduledInterview> possibleInterviews =
         ShadowLoadInterviewsServlet.getPossibleInterviews(
             scheduledInterviewDao, selectedPosition, interviewRange, personDao, shadowId);
+
     Set<ScheduledInterview> notValidInterviews = new HashSet<ScheduledInterview>();
     // We want to remove all interviews where the company or job does not match that
     // specified in the request.
-    String interviewerCompany = putRequest.getCompany();
-    String interviewerJob = putRequest.getJob();
```

Comment:
Please put these back where they were, Alex wants them as close to where they're used as possible.

Reply:
Done